pkg/app/app.go

```diff
@@ -57,12 +57,12 @@ func RealMain(opts types.Options, d types.Deployer, tester types.Tester) (result
 	// TODO(bentheelder): signal handling & timeout
 
 	// ensure the artifacts dir
-	if err := os.MkdirAll(opts.ArtifactsDir(), os.ModePerm); err != nil {
+	if err := os.MkdirAll(opts.RunDir(), os.ModePerm); err != nil {
```

Comment:
nit: since we are already changing stuff related to this 
let's `klog.Infof(opts.RunDir())` similar to how we log the runID on lin 87 for consistency

Reply:
Good idea - done!